test/Dialect/FIRRTL/errors.mlir

```diff
@@ -1184,6 +1184,30 @@ firrtl.circuit "NoDefineIntoRefSub" {
   }
 }
 
+// -----
+// Check that you can't drive a source.
+
+firrtl.circuit "PropertyDriveSource" {
+  // @expected-note @below {{the destination was defined here}}
+  firrtl.module @PropertyDriveSource(in %in: !firrtl.string) {
+    %0 = firrtl.string "hello"
+    // expected-error @below {{connect has invalid flow: the destination expression "in" has source flow, expected sink or duplex flow}}
+    firrtl.propassign %in, %0 : !firrtl.string
+  }
+}
+
+// -----
+// Check that you can't drive a sink more than once.
+
+firrtl.circuit "PropertyDoubleDrive" {
```

Comment:
Consider adding test for the flow-checking portion?  Input properties seem likely to be fairly common (?) ?.

Reply:
*blinks a few times* that's exactly what the test above does, my mistake!